  INFO root [0x0000222c] [2018-05-06 13:45:37,264] - 
  INFO root [0x0000222c] [2018-05-06 13:45:37,393] - -----------------------------------------------------------------------------------------


  INFO root [0x0000222c] [2018-05-06 13:45:39,187] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x0000222c] [2018-05-06 13:45:39,188] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x0000222c] [2018-05-06 13:45:39,188] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x0000222c] [2018-05-06 13:45:39,999] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x0000222c] [2018-05-06 13:45:40,070] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x0000222c] [2018-05-06 13:45:40,075] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50293 )
  INFO root [0x0000222c] [2018-05-06 13:45:40,075] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50293, SOMAXCONN=128.
  INFO root [0x0000222c] [2018-05-06 13:45:40,076] - server-configs:
  INFO root [0x0000222c] [2018-05-06 13:45:40,076] - 	gameUpdateHertz : 10
  INFO root [0x0000222c] [2018-05-06 13:45:40,077] - 	entryScriptFile : kbemain
  INFO root [0x0000222c] [2018-05-06 13:45:40,077] - 	internalAddr : 192.168.0.106:50293
  INFO root [0x0000222c] [2018-05-06 13:45:40,077] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x0000222c] [2018-05-06 13:45:40,078] - 	componentID : 7001
  INFO root [0x0000222c] [2018-05-06 13:45:40,998] - Components::findLogger: found logger, addr:192.168.0.106:50274
  INFO root [0x0000222c] [2018-05-06 13:45:40,998] - Components::addComponent: reset orderLog, uid=4706!
  INFO root [0x0000222c] [2018-05-06 13:45:40,998] - Components::addComponent[logger], uid=4706, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000222c] [2018-05-06 13:45:41,012] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 4706. PID: 10564 ----
  INFO root [0x0000222c] [2018-05-06 13:45:41,012] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x0000222c] [2018-05-06 13:45:41,013] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000222c] [2018-05-06 13:45:41,042] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,457] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,465] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,481] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,551] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,553] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,554] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,596] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,653] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000222c] [2018-05-06 13:45:41,700] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x0000222c] [2018-05-06 13:45:41,700] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,791] - loaded entity-script:Account(1).
 DEBUG root [0x0000222c] [2018-05-06 13:45:41,799] - loaded entity-script:CheckName(2).
  INFO root [0x0000222c] [2018-05-06 13:45:41,799] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000222c] [2018-05-06 13:45:41,800] - TelnetServer server is running on port 40000
  INFO root [0x0000222c] [2018-05-06 13:45:42,133] - ---- baseapp is running ----
 DEBUG root [0x0000222c] [2018-05-06 13:45:42,182] - Components::process(): Request for the process of identity...
  WARN root [0x0000222c] [2018-05-06 13:45:48,390] - Forwarding logs to logger[192.168.0.106:50274/0/0/0]...
 ERROR root [0x0000222c] [2018-05-06 13:46:05,863] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x0000222c] [2018-05-06 13:46:05,873] - The following logs sent to logger failed:
  WARN root [0x0000222c] [2018-05-06 13:46:05,873] - ==> [2018-05-06 13:46:05 859] TCPPacketReceiver::processPendingEvents(192.168.0.106:50274): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000222c] [2018-05-06 13:46:05,874] - ==> [2018-05-06 13:46:05 864] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x0000222c] [2018-05-06 13:46:05,916] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50277): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000222c] [2018-05-06 13:46:05,921] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x0000222c] [2018-05-06 13:46:05,978] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x0000222c] [2018-05-06 13:46:05,979] - onBaseAppShutDown: state=0
  INFO root [0x0000222c] [2018-05-06 13:46:05,979] - Shutdowner::onShutdown: shutting down
S_INFO root [0x0000222c] [2018-05-06 13:46:05,979] - onBaseAppShutDown: state=1
  INFO root [0x0000222c] [2018-05-06 13:46:05,979] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x0000222c] [2018-05-06 13:46:05,980] - onBaseAppShutDown: state=2
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,057] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,058] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,058] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,058] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,058] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,058] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,058] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000222c] [2018-05-06 13:46:06,058] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00000d54] [2018-05-06 13:46:07,119] - 
  INFO root [0x00000d54] [2018-05-06 13:46:07,120] - -----------------------------------------------------------------------------------------


  INFO root [0x00000d54] [2018-05-06 13:46:07,120] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00000d54] [2018-05-06 13:46:07,120] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00000d54] [2018-05-06 13:46:07,121] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00000d54] [2018-05-06 13:46:07,132] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00000d54] [2018-05-06 13:46:07,132] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00000d54] [2018-05-06 13:46:07,135] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50314 )
  INFO root [0x00000d54] [2018-05-06 13:46:07,135] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50314, SOMAXCONN=128.
  INFO root [0x00000d54] [2018-05-06 13:46:07,136] - server-configs:
  INFO root [0x00000d54] [2018-05-06 13:46:07,136] - 	gameUpdateHertz : 10
  INFO root [0x00000d54] [2018-05-06 13:46:07,136] - 	entryScriptFile : kbemain
  INFO root [0x00000d54] [2018-05-06 13:46:07,136] - 	internalAddr : 192.168.0.106:50314
  INFO root [0x00000d54] [2018-05-06 13:46:07,136] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00000d54] [2018-05-06 13:46:07,136] - 	componentID : 7001
  INFO root [0x00000d54] [2018-05-06 13:46:08,840] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 4915. PID: 5876 ----
  INFO root [0x00000d54] [2018-05-06 13:46:08,841] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00000d54] [2018-05-06 13:46:08,841] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000d54] [2018-05-06 13:46:08,842] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,966] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,966] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,967] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,970] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,970] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,970] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,970] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000d54] [2018-05-06 13:46:08,997] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000d54] [2018-05-06 13:46:09,025] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00000d54] [2018-05-06 13:46:09,026] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000d54] [2018-05-06 13:46:09,035] - loaded entity-script:Account(1).
 DEBUG root [0x00000d54] [2018-05-06 13:46:09,036] - loaded entity-script:CheckName(2).
  INFO root [0x00000d54] [2018-05-06 13:46:09,036] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000d54] [2018-05-06 13:46:09,036] - TelnetServer server is running on port 40000
  INFO root [0x00000d54] [2018-05-06 13:46:09,336] - ---- baseapp is running ----
 DEBUG root [0x00000d54] [2018-05-06 13:46:09,336] - Components::process(): Request for the process of identity...
  WARN root [0x00000d54] [2018-05-06 13:46:15,410] - EntityApp::checkTickPeriod: tick took 6073% (6.07 seconds)!
 DEBUG root [0x00000d54] [2018-05-06 13:46:15,640] - Baseapp::lookApp: 192.168.0.106:50330/0/0/0
  INFO root [0x00000d54] [2018-05-06 13:46:16,424] - Components::findComponents: find logger(1)...
  WARN root [0x00000d54] [2018-05-06 13:46:16,735] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20867, errorno=10048
  WARN root [0x00000d54] [2018-05-06 13:46:16,736] - EntityApp::checkTickPeriod: tick took 398% (0.40 seconds)!
  INFO root [0x00000d54] [2018-05-06 13:46:17,737] - Components::findComponents: find logger(2)...
  INFO root [0x00000d54] [2018-05-06 13:46:17,785] - Components::findComponents: found logger, addr:192.168.0.106:50316
  INFO root [0x00000d54] [2018-05-06 13:46:17,786] - Components::addComponent: reset orderLog, uid=4915!
  INFO root [0x00000d54] [2018-05-06 13:46:17,786] - Components::addComponent[logger], uid=4915, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000d54] [2018-05-06 13:46:17,788] - Components::findComponents: find dbmgr(3)...
  INFO root [0x00000d54] [2018-05-06 13:46:17,888] - Components::findComponents: found dbmgr, addr:192.168.0.106:50310
  INFO root [0x00000d54] [2018-05-06 13:46:17,888] - Components::addComponent: reset orderLog, uid=4915!
  INFO root [0x00000d54] [2018-05-06 13:46:17,888] - Components::addComponent[dbmgr], uid=4915, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000d54] [2018-05-06 13:46:17,889] - Components::findComponents: found dbmgr, addr:192.168.0.106:50310
  WARN root [0x00000d54] [2018-05-06 13:46:18,889] - EntityApp::checkTickPeriod: tick took 1252% (1.25 seconds)!
  WARN root [0x00000d54] [2018-05-06 13:46:18,891] - Forwarding logs to logger[192.168.0.106:50316/0/0/0]...
 ERROR root [0x00000d54] [2018-05-06 13:51:34,946] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00000d54] [2018-05-06 13:51:34,958] - The following logs sent to logger failed:
  WARN root [0x00000d54] [2018-05-06 13:51:34,958] - ==> [2018-05-06 13:51:34 945] TCPPacketReceiver::processPendingEvents(192.168.0.106:50316): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000d54] [2018-05-06 13:51:34,958] - ==> [2018-05-06 13:51:34 946] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00000d54] [2018-05-06 13:51:35,004] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50310): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000d54] [2018-05-06 13:51:35,007] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00000d54] [2018-05-06 13:51:35,013] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00000d54] [2018-05-06 13:51:35,015] - onBaseAppShutDown: state=0
  INFO root [0x00000d54] [2018-05-06 13:51:35,015] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00000d54] [2018-05-06 13:51:35,015] - onBaseAppShutDown: state=1
  INFO root [0x00000d54] [2018-05-06 13:51:35,015] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000d54] [2018-05-06 13:51:35,016] - onBaseAppShutDown: state=2
 ERROR root [0x00000d54] [2018-05-06 13:51:35,024] - Account::onCellWriteToDBCompleted(2001): not found dbmgr!
 ERROR root [0x00000d54] [2018-05-06 13:51:35,025] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00000d54] [2018-05-06 13:51:35,025] - CheckName::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00000d54] [2018-05-06 13:51:35,025] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,026] - Account::~Account(): 2001
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,026] - CheckName::~CheckName(): 2002
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,026] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,027] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,027] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,027] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,027] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,027] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,027] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,027] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00000d54] [2018-05-06 13:51:35,035] - Script::uninstall(): is successfully!
 DEBUG root [0x00000d54] [2018-05-06 13:51:35,035] - ThreadPool::destroy(): starting size 2.
  INFO root [0x0000241c] [2018-05-06 13:51:36,123] - 
  INFO root [0x0000241c] [2018-05-06 13:51:36,123] - -----------------------------------------------------------------------------------------


  INFO root [0x0000241c] [2018-05-06 13:51:36,123] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x0000241c] [2018-05-06 13:51:36,123] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x0000241c] [2018-05-06 13:51:36,124] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x0000241c] [2018-05-06 13:51:36,126] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x0000241c] [2018-05-06 13:51:36,126] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50403 )
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50403, SOMAXCONN=128.
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - server-configs:
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - 	gameUpdateHertz : 10
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - 	entryScriptFile : kbemain
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - 	internalAddr : 192.168.0.106:50403
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x0000241c] [2018-05-06 13:51:36,128] - 	componentID : 7001
  WARN root [0x0000241c] [2018-05-06 13:51:36,636] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20936, errorno=10048
  INFO root [0x0000241c] [2018-05-06 13:51:36,639] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 5989. PID: 3972 ----
  INFO root [0x0000241c] [2018-05-06 13:51:36,640] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x0000241c] [2018-05-06 13:51:36,640] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000241c] [2018-05-06 13:51:36,640] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,763] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,764] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,764] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,767] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,767] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,767] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,768] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,793] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000241c] [2018-05-06 13:51:36,820] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x0000241c] [2018-05-06 13:51:36,820] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,829] - loaded entity-script:Account(1).
 DEBUG root [0x0000241c] [2018-05-06 13:51:36,830] - loaded entity-script:CheckName(2).
  INFO root [0x0000241c] [2018-05-06 13:51:36,830] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000241c] [2018-05-06 13:51:36,830] - TelnetServer server is running on port 40000
  INFO root [0x0000241c] [2018-05-06 13:51:37,108] - ---- baseapp is running ----
 DEBUG root [0x0000241c] [2018-05-06 13:51:37,108] - Components::process(): Request for the process of identity...
  WARN root [0x0000241c] [2018-05-06 13:51:43,167] - EntityApp::checkTickPeriod: tick took 6058% (6.06 seconds)!
  INFO root [0x0000241c] [2018-05-06 13:51:44,209] - Components::findComponents: find logger(1)...
  INFO root [0x0000241c] [2018-05-06 13:51:44,271] - Components::findComponents: found logger, addr:192.168.0.106:50400
  INFO root [0x0000241c] [2018-05-06 13:51:44,271] - Components::addComponent: reset orderLog, uid=5989!
  INFO root [0x0000241c] [2018-05-06 13:51:44,271] - Components::addComponent[logger], uid=5989, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000241c] [2018-05-06 13:51:44,272] - Components::findComponents: find dbmgr(2)...
  WARN root [0x0000241c] [2018-05-06 13:51:45,375] - EntityApp::checkTickPeriod: tick took 1266% (1.27 seconds)!
  WARN root [0x0000241c] [2018-05-06 13:51:45,376] - Forwarding logs to logger[192.168.0.106:50400/0/0/0]...
 ERROR root [0x0000241c] [2018-05-06 13:51:50,373] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x0000241c] [2018-05-06 13:51:50,380] - The following logs sent to logger failed:
  WARN root [0x0000241c] [2018-05-06 13:51:50,380] - ==> [2018-05-06 13:51:50 373] TCPPacketReceiver::processPendingEvents(192.168.0.106:50400): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000241c] [2018-05-06 13:51:50,380] - ==> [2018-05-06 13:51:50 374] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x0000241c] [2018-05-06 13:51:50,525] - Components::findComponents: find dbmgr(5)...
  INFO root [0x00001d58] [2018-05-06 13:51:51,525] - 
  INFO root [0x00001d58] [2018-05-06 13:51:51,525] - -----------------------------------------------------------------------------------------


  INFO root [0x00001d58] [2018-05-06 13:51:51,526] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001d58] [2018-05-06 13:51:51,526] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001d58] [2018-05-06 13:51:51,526] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001d58] [2018-05-06 13:51:51,533] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00001d58] [2018-05-06 13:51:51,533] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00001d58] [2018-05-06 13:51:51,538] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50427 )
  INFO root [0x00001d58] [2018-05-06 13:51:51,538] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50427, SOMAXCONN=128.
  INFO root [0x00001d58] [2018-05-06 13:51:51,538] - server-configs:
  INFO root [0x00001d58] [2018-05-06 13:51:51,539] - 	gameUpdateHertz : 10
  INFO root [0x00001d58] [2018-05-06 13:51:51,539] - 	entryScriptFile : kbemain
  INFO root [0x00001d58] [2018-05-06 13:51:51,539] - 	internalAddr : 192.168.0.106:50427
  INFO root [0x00001d58] [2018-05-06 13:51:51,539] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00001d58] [2018-05-06 13:51:51,539] - 	componentID : 7001
  INFO root [0x00001d58] [2018-05-06 13:51:53,229] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 6042. PID: 4596 ----
  INFO root [0x00001d58] [2018-05-06 13:51:53,229] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001d58] [2018-05-06 13:51:53,230] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001d58] [2018-05-06 13:51:53,230] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,358] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,359] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,359] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,362] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,362] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,362] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,363] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,391] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001d58] [2018-05-06 13:51:53,428] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001d58] [2018-05-06 13:51:53,428] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,438] - loaded entity-script:Account(1).
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,439] - loaded entity-script:CheckName(2).
  INFO root [0x00001d58] [2018-05-06 13:51:53,439] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001d58] [2018-05-06 13:51:53,439] - TelnetServer server is running on port 40000
  INFO root [0x00001d58] [2018-05-06 13:51:53,765] - ---- baseapp is running ----
 DEBUG root [0x00001d58] [2018-05-06 13:51:53,766] - Components::process(): Request for the process of identity...
  WARN root [0x00001d58] [2018-05-06 13:51:59,851] - EntityApp::checkTickPeriod: tick took 6085% (6.09 seconds)!
  INFO root [0x00001d58] [2018-05-06 13:52:00,928] - Components::findComponents: find logger(1)...
  INFO root [0x00001d58] [2018-05-06 13:52:00,938] - Components::findComponents: found logger, addr:192.168.0.106:50428
  INFO root [0x00001d58] [2018-05-06 13:52:00,939] - Components::addComponent: reset orderLog, uid=6042!
  INFO root [0x00001d58] [2018-05-06 13:52:00,939] - Components::addComponent[logger], uid=6042, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d58] [2018-05-06 13:52:00,941] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00001d58] [2018-05-06 13:52:02,053] - Forwarding logs to logger[192.168.0.106:50428/0/0/0]...
 ERROR root [0x00001d58] [2018-05-06 13:52:04,497] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001d58] [2018-05-06 13:52:04,503] - The following logs sent to logger failed:
  WARN root [0x00001d58] [2018-05-06 13:52:04,503] - ==> [2018-05-06 13:52:04 494] TCPPacketReceiver::processPendingEvents(192.168.0.106:50428): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001d58] [2018-05-06 13:52:04,503] - ==> [2018-05-06 13:52:04 498] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000026a0] [2018-05-06 13:52:05,633] - 
  INFO root [0x000026a0] [2018-05-06 13:52:05,634] - -----------------------------------------------------------------------------------------


  INFO root [0x000026a0] [2018-05-06 13:52:05,634] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x000026a0] [2018-05-06 13:52:05,635] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x000026a0] [2018-05-06 13:52:05,635] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x000026a0] [2018-05-06 13:52:05,640] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x000026a0] [2018-05-06 13:52:05,641] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x000026a0] [2018-05-06 13:52:05,647] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50449 )
  INFO root [0x000026a0] [2018-05-06 13:52:05,647] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50449, SOMAXCONN=128.
  INFO root [0x000026a0] [2018-05-06 13:52:05,647] - server-configs:
  INFO root [0x000026a0] [2018-05-06 13:52:05,647] - 	gameUpdateHertz : 10
  INFO root [0x000026a0] [2018-05-06 13:52:05,647] - 	entryScriptFile : kbemain
  INFO root [0x000026a0] [2018-05-06 13:52:05,648] - 	internalAddr : 192.168.0.106:50449
  INFO root [0x000026a0] [2018-05-06 13:52:05,648] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x000026a0] [2018-05-06 13:52:05,648] - 	componentID : 7001
  INFO root [0x000026a0] [2018-05-06 13:52:07,349] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 6087. PID: 8180 ----
  INFO root [0x000026a0] [2018-05-06 13:52:07,350] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x000026a0] [2018-05-06 13:52:07,350] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000026a0] [2018-05-06 13:52:07,350] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,479] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,479] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,479] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,482] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,483] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,483] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,483] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,519] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000026a0] [2018-05-06 13:52:07,554] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000026a0] [2018-05-06 13:52:07,554] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,566] - loaded entity-script:Account(1).
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,567] - loaded entity-script:CheckName(2).
  INFO root [0x000026a0] [2018-05-06 13:52:07,567] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000026a0] [2018-05-06 13:52:07,568] - TelnetServer server is running on port 40000
  INFO root [0x000026a0] [2018-05-06 13:52:07,898] - ---- baseapp is running ----
 DEBUG root [0x000026a0] [2018-05-06 13:52:07,898] - Components::process(): Request for the process of identity...
  WARN root [0x000026a0] [2018-05-06 13:52:11,249] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20677, errorno=10048
  WARN root [0x000026a0] [2018-05-06 13:52:14,318] - EntityApp::checkTickPeriod: tick took 6420% (6.42 seconds)!
  INFO root [0x000026a0] [2018-05-06 13:52:15,398] - Components::findComponents: find logger(1)...
  INFO root [0x000026a0] [2018-05-06 13:52:15,481] - Components::findComponents: found logger, addr:192.168.0.106:50447
  INFO root [0x000026a0] [2018-05-06 13:52:15,481] - Components::addComponent: reset orderLog, uid=6087!
  INFO root [0x000026a0] [2018-05-06 13:52:15,481] - Components::addComponent[logger], uid=6087, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000026a0] [2018-05-06 13:52:15,482] - Components::findComponents: find dbmgr(2)...
  WARN root [0x000026a0] [2018-05-06 13:52:16,585] - EntityApp::checkTickPeriod: tick took 1287% (1.29 seconds)!
  WARN root [0x000026a0] [2018-05-06 13:52:16,587] - Forwarding logs to logger[192.168.0.106:50447/0/0/0]...
 ERROR root [0x000026a0] [2018-05-06 13:52:42,182] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000026a0] [2018-05-06 13:52:42,189] - The following logs sent to logger failed:
  WARN root [0x000026a0] [2018-05-06 13:52:42,189] - ==> [2018-05-06 13:52:42 180] TCPPacketReceiver::processPendingEvents(192.168.0.106:50447): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000026a0] [2018-05-06 13:52:42,189] - ==> [2018-05-06 13:52:42 182] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000013d4] [2018-05-06 13:53:15,806] - 
  INFO root [0x000013d4] [2018-05-06 13:53:15,807] - -----------------------------------------------------------------------------------------


  INFO root [0x000013d4] [2018-05-06 13:53:15,808] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x000013d4] [2018-05-06 13:53:15,808] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x000013d4] [2018-05-06 13:53:15,808] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x000013d4] [2018-05-06 13:53:15,815] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x000013d4] [2018-05-06 13:53:15,815] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x000013d4] [2018-05-06 13:53:15,818] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50519 )
  INFO root [0x000013d4] [2018-05-06 13:53:15,818] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50519, SOMAXCONN=128.
  INFO root [0x000013d4] [2018-05-06 13:53:15,818] - server-configs:
  INFO root [0x000013d4] [2018-05-06 13:53:15,818] - 	gameUpdateHertz : 10
  INFO root [0x000013d4] [2018-05-06 13:53:15,818] - 	entryScriptFile : kbemain
  INFO root [0x000013d4] [2018-05-06 13:53:15,818] - 	internalAddr : 192.168.0.106:50519
  INFO root [0x000013d4] [2018-05-06 13:53:15,819] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x000013d4] [2018-05-06 13:53:15,819] - 	componentID : 7001
  INFO root [0x000013d4] [2018-05-06 13:53:17,517] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 6316. PID: 5616 ----
  INFO root [0x000013d4] [2018-05-06 13:53:17,518] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x000013d4] [2018-05-06 13:53:17,518] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000013d4] [2018-05-06 13:53:17,518] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,649] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,649] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,650] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,652] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,653] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,653] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,653] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,684] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000013d4] [2018-05-06 13:53:17,711] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000013d4] [2018-05-06 13:53:17,711] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,724] - loaded entity-script:Account(1).
 DEBUG root [0x000013d4] [2018-05-06 13:53:17,725] - loaded entity-script:CheckName(2).
  INFO root [0x000013d4] [2018-05-06 13:53:17,725] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000013d4] [2018-05-06 13:53:17,726] - TelnetServer server is running on port 40000
  INFO root [0x000013d4] [2018-05-06 13:53:18,081] - ---- baseapp is running ----
 DEBUG root [0x000013d4] [2018-05-06 13:53:18,082] - Components::process(): Request for the process of identity...
  WARN root [0x000013d4] [2018-05-06 13:53:24,134] - EntityApp::checkTickPeriod: tick took 6053% (6.05 seconds)!
  INFO root [0x000013d4] [2018-05-06 13:53:25,182] - Components::findComponents: find logger(1)...
  INFO root [0x000013d4] [2018-05-06 13:53:25,231] - Components::findComponents: found logger, addr:192.168.0.106:50515
  INFO root [0x000013d4] [2018-05-06 13:53:25,232] - Components::addComponent: reset orderLog, uid=6316!
  INFO root [0x000013d4] [2018-05-06 13:53:25,232] - Components::addComponent[logger], uid=6316, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000013d4] [2018-05-06 13:53:25,233] - Components::findComponents: find dbmgr(2)...
  WARN root [0x000013d4] [2018-05-06 13:53:26,340] - EntityApp::checkTickPeriod: tick took 1258% (1.26 seconds)!
  WARN root [0x000013d4] [2018-05-06 13:53:26,342] - Forwarding logs to logger[192.168.0.106:50515/0/0/0]...
 ERROR root [0x000013d4] [2018-05-06 13:54:14,392] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000013d4] [2018-05-06 13:54:14,397] - The following logs sent to logger failed:
  WARN root [0x000013d4] [2018-05-06 13:54:14,397] - ==> [2018-05-06 13:54:14 392] TCPPacketReceiver::processPendingEvents(192.168.0.106:50515): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000013d4] [2018-05-06 13:54:14,398] - ==> [2018-05-06 13:54:14 393] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002f30] [2018-05-06 13:56:01,794] - 
  INFO root [0x00002f30] [2018-05-06 13:56:01,794] - -----------------------------------------------------------------------------------------


  INFO root [0x00002f30] [2018-05-06 13:56:01,795] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00002f30] [2018-05-06 13:56:01,795] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00002f30] [2018-05-06 13:56:01,795] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00002f30] [2018-05-06 13:56:01,798] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00002f30] [2018-05-06 13:56:01,798] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00002f30] [2018-05-06 13:56:01,799] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50590 )
  INFO root [0x00002f30] [2018-05-06 13:56:01,799] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50590, SOMAXCONN=128.
  INFO root [0x00002f30] [2018-05-06 13:56:01,799] - server-configs:
  INFO root [0x00002f30] [2018-05-06 13:56:01,799] - 	gameUpdateHertz : 10
  INFO root [0x00002f30] [2018-05-06 13:56:01,800] - 	entryScriptFile : kbemain
  INFO root [0x00002f30] [2018-05-06 13:56:01,800] - 	internalAddr : 192.168.0.106:50590
  INFO root [0x00002f30] [2018-05-06 13:56:01,800] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00002f30] [2018-05-06 13:56:01,800] - 	componentID : 7001
  INFO root [0x00002f30] [2018-05-06 13:56:03,487] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 6858. PID: 8968 ----
  INFO root [0x00002f30] [2018-05-06 13:56:03,488] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00002f30] [2018-05-06 13:56:03,488] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002f30] [2018-05-06 13:56:03,488] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,612] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,613] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,613] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,616] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,616] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,616] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,617] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,645] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002f30] [2018-05-06 13:56:03,677] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002f30] [2018-05-06 13:56:03,677] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,689] - loaded entity-script:Account(1).
 DEBUG root [0x00002f30] [2018-05-06 13:56:03,690] - loaded entity-script:CheckName(2).
  INFO root [0x00002f30] [2018-05-06 13:56:03,690] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002f30] [2018-05-06 13:56:03,691] - TelnetServer server is running on port 40000
  INFO root [0x00002f30] [2018-05-06 13:56:04,017] - ---- baseapp is running ----
 DEBUG root [0x00002f30] [2018-05-06 13:56:04,018] - Components::process(): Request for the process of identity...
  WARN root [0x00002f30] [2018-05-06 13:56:10,069] - EntityApp::checkTickPeriod: tick took 6051% (6.05 seconds)!
  INFO root [0x00002f30] [2018-05-06 13:56:11,097] - Components::findComponents: find logger(1)...
  INFO root [0x00002f30] [2018-05-06 13:56:11,198] - Components::findComponents: found logger, addr:192.168.0.106:50586
  INFO root [0x00002f30] [2018-05-06 13:56:11,198] - Components::addComponent: reset orderLog, uid=6858!
  INFO root [0x00002f30] [2018-05-06 13:56:11,199] - Components::addComponent[logger], uid=6858, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002f30] [2018-05-06 13:56:11,202] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002f30] [2018-05-06 13:56:11,306] - Components::findComponents: found dbmgr, addr:192.168.0.106:50585
  INFO root [0x00002f30] [2018-05-06 13:56:11,306] - Components::addComponent: reset orderLog, uid=6858!
  INFO root [0x00002f30] [2018-05-06 13:56:11,306] - Components::addComponent[dbmgr], uid=6858, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002f30] [2018-05-06 13:56:11,307] - Components::findComponents: found dbmgr, addr:192.168.0.106:50585
  WARN root [0x00002f30] [2018-05-06 13:56:12,307] - Forwarding logs to logger[192.168.0.106:50586/0/0/0]...
 ERROR root [0x00002f30] [2018-05-06 13:56:42,787] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002f30] [2018-05-06 13:56:42,798] - The following logs sent to logger failed:
  WARN root [0x00002f30] [2018-05-06 13:56:42,799] - ==> [2018-05-06 13:56:42 781] TCPPacketReceiver::processPendingEvents(192.168.0.106:50586): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002f30] [2018-05-06 13:56:42,799] - ==> [2018-05-06 13:56:42 787] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002f30] [2018-05-06 13:56:42,839] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50585): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002f30] [2018-05-06 13:56:42,841] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002f30] [2018-05-06 13:56:42,853] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002f30] [2018-05-06 13:56:42,854] - onBaseAppShutDown: state=0
  INFO root [0x00002f30] [2018-05-06 13:56:42,854] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002f30] [2018-05-06 13:56:42,854] - onBaseAppShutDown: state=1
  INFO root [0x00002f30] [2018-05-06 13:56:42,854] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002f30] [2018-05-06 13:56:42,854] - onBaseAppShutDown: state=2
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,862] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,862] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,862] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,862] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,863] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,863] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,863] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,863] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002f30] [2018-05-06 13:56:42,870] - Script::uninstall(): is successfully!
 DEBUG root [0x00002f30] [2018-05-06 13:56:42,870] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001164] [2018-05-06 13:56:44,016] - 
  INFO root [0x00001164] [2018-05-06 13:56:44,016] - -----------------------------------------------------------------------------------------


  INFO root [0x00001164] [2018-05-06 13:56:44,017] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001164] [2018-05-06 13:56:44,017] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001164] [2018-05-06 13:56:44,017] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001164] [2018-05-06 13:56:44,019] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00001164] [2018-05-06 13:56:44,019] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00001164] [2018-05-06 13:56:44,020] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50628 )
  INFO root [0x00001164] [2018-05-06 13:56:44,020] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50628, SOMAXCONN=128.
  INFO root [0x00001164] [2018-05-06 13:56:44,020] - server-configs:
  INFO root [0x00001164] [2018-05-06 13:56:44,020] - 	gameUpdateHertz : 10
  INFO root [0x00001164] [2018-05-06 13:56:44,020] - 	entryScriptFile : kbemain
  INFO root [0x00001164] [2018-05-06 13:56:44,021] - 	internalAddr : 192.168.0.106:50628
  INFO root [0x00001164] [2018-05-06 13:56:44,021] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00001164] [2018-05-06 13:56:44,021] - 	componentID : 7001
  INFO root [0x00001164] [2018-05-06 13:56:44,491] - Components::findLogger: found logger, addr:192.168.0.106:50625
  INFO root [0x00001164] [2018-05-06 13:56:44,491] - Components::addComponent: reset orderLog, uid=6995!
  INFO root [0x00001164] [2018-05-06 13:56:44,491] - Components::addComponent[logger], uid=6995, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001164] [2018-05-06 13:56:44,496] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 6995. PID: 5276 ----
  INFO root [0x00001164] [2018-05-06 13:56:44,496] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001164] [2018-05-06 13:56:44,496] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001164] [2018-05-06 13:56:44,497] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001164] [2018-05-06 13:56:44,619] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001164] [2018-05-06 13:56:44,619] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001164] [2018-05-06 13:56:44,620] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001164] [2018-05-06 13:56:44,622] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001164] [2018-05-06 13:56:44,623] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001164] [2018-05-06 13:56:44,623] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001164] [2018-05-06 13:56:44,623] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001164] [2018-05-06 13:56:44,650] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001164] [2018-05-06 13:56:44,680] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001164] [2018-05-06 13:56:44,681] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001164] [2018-05-06 13:56:44,691] - loaded entity-script:Account(1).
 DEBUG root [0x00001164] [2018-05-06 13:56:44,692] - loaded entity-script:CheckName(2).
  INFO root [0x00001164] [2018-05-06 13:56:44,692] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001164] [2018-05-06 13:56:44,692] - TelnetServer server is running on port 40000
  INFO root [0x00001164] [2018-05-06 13:56:45,005] - ---- baseapp is running ----
 DEBUG root [0x00001164] [2018-05-06 13:56:45,005] - Components::process(): Request for the process of identity...
  WARN root [0x00001164] [2018-05-06 13:56:51,075] - Forwarding logs to logger[192.168.0.106:50625/0/0/0]...
 ERROR root [0x00001164] [2018-05-06 14:06:40,558] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001164] [2018-05-06 14:06:40,564] - The following logs sent to logger failed:
  WARN root [0x00001164] [2018-05-06 14:06:40,564] - ==> [2018-05-06 14:06:40 554] TCPPacketReceiver::processPendingEvents(192.168.0.106:50625): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001164] [2018-05-06 14:06:40,564] - ==> [2018-05-06 14:06:40 558] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001164] [2018-05-06 14:06:40,608] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50621): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001164] [2018-05-06 14:06:40,611] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001164] [2018-05-06 14:06:40,616] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001164] [2018-05-06 14:06:40,617] - onBaseAppShutDown: state=0
  INFO root [0x00001164] [2018-05-06 14:06:40,617] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001164] [2018-05-06 14:06:40,617] - onBaseAppShutDown: state=1
  INFO root [0x00001164] [2018-05-06 14:06:40,618] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001164] [2018-05-06 14:06:40,618] - onBaseAppShutDown: state=2
 ERROR root [0x00001164] [2018-05-06 14:06:40,625] - CheckName::onCellWriteToDBCompleted(4): not found dbmgr!
 ERROR root [0x00001164] [2018-05-06 14:06:40,626] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001164] [2018-05-06 14:06:40,626] - CheckName::onCellWriteToDBCompleted(6): not found dbmgr!
 ERROR root [0x00001164] [2018-05-06 14:06:40,627] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001164] [2018-05-06 14:06:40,627] - Account::onCellWriteToDBCompleted(9): not found dbmgr!
 ERROR root [0x00001164] [2018-05-06 14:06:40,627] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00001164] [2018-05-06 14:06:40,628] - CheckName::~CheckName(): 4
 DEBUG root [0x00001164] [2018-05-06 14:06:40,628] - CheckName::~CheckName(): 6
 DEBUG root [0x00001164] [2018-05-06 14:06:40,628] - Account::~Account(): 9
 DEBUG root [0x00001164] [2018-05-06 14:06:40,628] - CheckName::~CheckName(): 10
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001164] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001164] [2018-05-06 14:06:40,638] - Script::uninstall(): is successfully!
 DEBUG root [0x00001164] [2018-05-06 14:06:40,638] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002470] [2018-05-06 14:06:41,755] - 
  INFO root [0x00002470] [2018-05-06 14:06:41,756] - -----------------------------------------------------------------------------------------


  INFO root [0x00002470] [2018-05-06 14:06:41,756] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00002470] [2018-05-06 14:06:41,757] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00002470] [2018-05-06 14:06:41,757] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00002470] [2018-05-06 14:06:41,760] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00002470] [2018-05-06 14:06:41,760] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00002470] [2018-05-06 14:06:41,761] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50829 )
  INFO root [0x00002470] [2018-05-06 14:06:41,762] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50829, SOMAXCONN=128.
  INFO root [0x00002470] [2018-05-06 14:06:41,762] - server-configs:
  INFO root [0x00002470] [2018-05-06 14:06:41,762] - 	gameUpdateHertz : 10
  INFO root [0x00002470] [2018-05-06 14:06:41,762] - 	entryScriptFile : kbemain
  INFO root [0x00002470] [2018-05-06 14:06:41,762] - 	internalAddr : 192.168.0.106:50829
  INFO root [0x00002470] [2018-05-06 14:06:41,762] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00002470] [2018-05-06 14:06:41,762] - 	componentID : 7001
  INFO root [0x00002470] [2018-05-06 14:06:43,482] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 8948. PID: 12600 ----
  INFO root [0x00002470] [2018-05-06 14:06:43,483] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00002470] [2018-05-06 14:06:43,483] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002470] [2018-05-06 14:06:43,483] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002470] [2018-05-06 14:06:43,611] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002470] [2018-05-06 14:06:43,612] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002470] [2018-05-06 14:06:43,612] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002470] [2018-05-06 14:06:43,616] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002470] [2018-05-06 14:06:43,616] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002470] [2018-05-06 14:06:43,617] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002470] [2018-05-06 14:06:43,618] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002470] [2018-05-06 14:06:43,654] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002470] [2018-05-06 14:06:43,693] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002470] [2018-05-06 14:06:43,693] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002470] [2018-05-06 14:06:43,713] - loaded entity-script:Account(1).
 DEBUG root [0x00002470] [2018-05-06 14:06:43,714] - loaded entity-script:CheckName(2).
  INFO root [0x00002470] [2018-05-06 14:06:43,714] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002470] [2018-05-06 14:06:43,715] - TelnetServer server is running on port 40000
  INFO root [0x00002470] [2018-05-06 14:06:44,041] - ---- baseapp is running ----
 DEBUG root [0x00002470] [2018-05-06 14:06:44,041] - Components::process(): Request for the process of identity...
  WARN root [0x00002470] [2018-05-06 14:06:50,098] - EntityApp::checkTickPeriod: tick took 6057% (6.06 seconds)!
  INFO root [0x00002470] [2018-05-06 14:06:51,141] - Components::findComponents: find logger(1)...
  INFO root [0x00002470] [2018-05-06 14:06:51,215] - Components::findComponents: found logger, addr:192.168.0.106:50825
  INFO root [0x00002470] [2018-05-06 14:06:51,216] - Components::addComponent: reset orderLog, uid=8948!
  INFO root [0x00002470] [2018-05-06 14:06:51,216] - Components::addComponent[logger], uid=8948, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002470] [2018-05-06 14:06:51,217] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002470] [2018-05-06 14:06:51,318] - Components::findComponents: found dbmgr, addr:192.168.0.106:50822
  INFO root [0x00002470] [2018-05-06 14:06:51,318] - Components::addComponent: reset orderLog, uid=8948!
  INFO root [0x00002470] [2018-05-06 14:06:51,319] - Components::addComponent[dbmgr], uid=8948, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002470] [2018-05-06 14:06:51,319] - Components::findComponents: found dbmgr, addr:192.168.0.106:50822
  WARN root [0x00002470] [2018-05-06 14:06:52,320] - EntityApp::checkTickPeriod: tick took 1278% (1.28 seconds)!
  WARN root [0x00002470] [2018-05-06 14:06:52,320] - Forwarding logs to logger[192.168.0.106:50825/0/0/0]...
